tools/jit/gen_jit_dispatch.py

```diff
@@ -144,7 +144,11 @@ def from_ivalue(arg, value):
         .layout(${layout})
         .device(${device})
         .pinned_memory(${pin_memory});
-auto result_ = torch::${name}(${args_with_tensor_options});
+#ifdef USE_STATIC_DISPATCH
```

Comment:
Is it to avoid creating variables? It might be better to put NoGrad guard on. Afaiu, we want to get rid of at:: factory functions eventually and just create always variables (cc @gchanan)

Reply:
Yeah it's to avoid creating variables. I don't think NoGrad works without additional changes, torch:: will always create a variable.

Reply:
I plan to look into optional build for autograd/variable functionality on top of this PR and make changes if I find anything - so it's fine as long as the static dispatching part works.